### ISSUE TITLE
Sample Project [Request]

### ISSUE THREAD

**Issue by austin-guevara** [salesforce-marketingcloud/blocksdk#1]
Would it be possible to include a folder with a super simple sample project? This would help devs to see the API in action.

Thanks!

**Comment by tbesluau** [comment]
We would probably make a different repo for it and link to it in the README. For now, please take a look at https://github.com/tbesluau/sampleblock for an example.

**Comment by alemieux3** [comment]
I agree with @austin-guevara I can't make heads or tails about what is absolutely necessary for this to work. After var sdk = new SDK(); I'm not sure what to do next? 

Also, when I install the example files locally, the UI shows up in blocktester, but it doesn't work. I get a console error Failed to load resource: the server responded with a status of 404 (Not Found) and it's looking for an SVG file here: http://localhost:3000/assets/icons/standard-sprite/svg/symbols.svg#contact

**Comment by ComputerTinker** [comment]
Here's a dead simple POC for those looking to wrap their heads around the different functions involved in this SDK:

```html
<html>
<head>
	<script src="../../js/blocksdk-1.1.1.js" type="text/javascript"></script>
</head>
<body>

<label for="cBtnText"><strong>Button Text:</strong></label><br>
<input type="text" id="cBtnText" value="Click Me" onkeyup="changeBtnParms();" style="width:100%;" /><br><br>

<script>
// Test changes to this block by using https://blocktester.herokuapp.com/

function restoreParms() {
	sdk.getData(function(objData) {
		if (objData.cBtnText)
			document.getElementById('cBtnText').value = objData.cBtnText;
	});
}

function saveParms() {
	objData = {
		cBtnText:  document.getElementById('cBtnText').value,
	};

	sdk.setData(objData, function(setData) {
	});
}

function changeBtnParms() {
	sdk.getContent(function(cContent) {
		cBtnText = document.getElementById('cBtnText').value;

		cContent = '<a href="#" style="background-color:#009900; color:#ffffff; display:inline-block; font-family:Helvetica,Arial,sans-serif; font-size:17px; font-weight:bold; text-align:center; text-decoration:none; padding:10px 20px;">' + cBtnText + '</a>';

		sdk.setContent(cContent, function(setContent) {
		});

		saveParms();	// Save options for next time
	});
}

var sdk = new window.sfdc.BlockSDK();
restoreParms();	// Restore last saved state
changeBtnParms();	// Force a render of the button
</script>
</body>
</html>
```

Hope this helps someone!

**Comment by alemieux3** [comment]
@ComputerTinker I put this file into the sampleblock-master dist folder and loaded it up in the blocktester and got this error: invalid block widget endpoint 

I notice that you are loading the blocksdk in the head with this line:
`<script src="../../js/blocksdk-1.1.1.js" type="text/javascript"></script>` 

Is this example not using NPM to install the block SDK library?

**Comment by alemieux3** [comment]
@ComputerTinker I downloaded the blocksdk.js from the repo and changed the reference in the HTML file. When I load it in blocktester, I no longer get the error, but I'm not sure what the app is supposed to do. I can change the text, but nothing is reflected in the block. Shouldn't the change be updated in the block?

**Comment by darleison-rodrigues-osf** [comment]
> We would probably make a different repo for it and link to it in the README. For now, please take a look at https://github.com/tbesluau/sampleblock for an example.

Hi All,
I started to build a 'custom content block reference' something like SiteGenesis or SFRA from Commerce Cloud. It has Travis CI/CD Integration, Heroku Deployment, Lightning Design System Component listing and more. I think it could leverage the development process with Content Blocks, because of theses features represent the best practices in software develpement. It still in development, all help will be appreciated https://github.com/darleison-rodrigues-osf/custom-content-block-reference.

**Comment by barcellos-pedro** [comment]
> @ComputerTinker I downloaded the blocksdk.js from the repo and changed the reference in the HTML file. When I load it in blocktester, I no longer get the error, but I'm not sure what the app is supposed to do. I can change the text, but nothing is reflected in the block. Shouldn't the change be updated in the block?

Hey, make sure the url its _**https**_ like `https://blocktester.herokuapp.com/`.
That worked for me. 

Also i made a [super simple project](https://github.com/barcellos-pedro/custom-block-widget-sample) so you can check it out.
